drivers/scheduler/k8s/specs/mysql/px-mysql-storage.yaml

```diff
@@ -42,7 +42,7 @@ metadata:
     name: px-mysql-sc-seq
 provisioner: kubernetes.io/portworx-volume
 parameters:
-   fs: "ext4"
+   fs: "xfs"
```

Comment:
@mborodin-px for now, can you also add `force_unsupported_fs_type: "true"` to the StorageClass? This will ensure xfs is exercise on existing PX versions.

Reply:
force_unsupported_fs_type should only be done for 2.0.x jobs for 2.1.x jobs